lightning/src/ln/channelmanager.rs

```diff
@@ -773,6 +779,9 @@ macro_rules! handle_error {
 							msg: update
 						});
 					}
+					if let Some(channel_id) = chan_id {
+						$self.pending_events.lock().unwrap().push(events::Event::ChannelClosed { channel_id,  err: ClosureDescriptor::ProcessingError });
```

Comment:
Can we include the error message in the `ClosureDescriptor::ProcessingError` enum variant?